osbs/build/build_requestv2.py

```diff
@@ -311,116 +298,18 @@ class BuildRequestV2(BaseBuildRequest):
     """
     Wraps logic for creating build inputs
     """
-    def __init__(self, build_json_store, outer_template=None, customize_conf=None):
+    def __init__(self, user_params, osbs_api, outer_template=None):
         """
         :param build_json_store: str, path to directory with JSON build files
         :param outer_template: str, path to outer template JSON
-        :param customize_conf: str, path to customize configuration JSON
         """
```

Comment:
I'd put assert/TypeError here when user_params don't match expected type
```
assert isinstance(user_params, BuildUserParams)
```
Simillar for source containers and base class

Reply:
Fixed.